src/components/sdk/MeetingControls/AudioInputControl.tsx

```diff
@@ -22,30 +21,31 @@ interface Props extends BaseSdkProps {
   mutedIconTitle?: string;
   /** Title attribute for the icon when unmuted, it defaults to `Microphone`. */
   unmutedIconTitle?: string;
-  /** A boolean that determines whether or not to include additional sample audio input devices, such as "None", "440 Hz". Defaults to true. This will be deprecated in the next major version. */
-  appendSampleDevices?: boolean;
 }
 
 const AudioInputControl: React.FC<Props> = ({
   muteLabel = 'Mute',
   unmuteLabel = 'Unmute',
   mutedIconTitle,
   unmutedIconTitle,
-  appendSampleDevices = true,
   ...rest
 }) => {
-  const meetingManager = useMeetingManager();
+  const selectAudioInput = useSelectAudioInputDevice();
```

Comment:
what is the difference between `AudioInputControl` and `MicSelection`?

Reply:
`MicSelection` is the one in Device Setup page.
`AudioInputControl` is the one in Meeting Control Bar.

Reply:
Yeah but can't tell from that name and the code is almost similar. So not sure why we need both (actually 3, `MicSelection`, `AudioInputControl`, and `AudioInputVFControl`.

Reply:
They are different UI.
`MicSelection` is `DeviceInput`(`FormField`)
`AudioInputControl` is` ControlBarButton`